src/tools/context/initial-context.ts

```diff
@@ -23,6 +23,12 @@ Collections organize content items into logical groups by team, brand, or projec
 ### Languages
 Languages define available locales for content. Each language can have a fallback language for content inheritance and can be activated or deactivated.
 
+### Roles
+Roles define permissions for users to access and manage content. Each role has a set of permissions that determine what actions a user of said role can perform. Project Manager is the predefined default role and is also the only role that has a codename.
```

Comment:
Initial context tool is being removed. Include the important information to tool descriptions.